README.md

```diff
@@ -149,6 +149,7 @@ When trying to optimize the time spent building the project, it is often helpful
 ## Maintainers
 - Jan Kossmann
 - Martin Boissier
+- Marcel Weisgut
```

Comment:
Do we have an order here? Looked like firstname. In this case you should be second.